delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/scalatest/ce/CatsIOValues.scala

```diff
@@ -12,8 +12,12 @@ trait CatsIOValues {
   self: Suite =>
 
   implicit final class CatsIOValuesOps[A](private val io: IO[A]) {
-    def accepted: A =
-      io.unsafeRunTimed(45.seconds).getOrElse(fail("IO timed out during .accepted call"))
+    def accepted(implicit pos: source.Position): A = {
+      io.attempt.unsafeRunTimed(45.seconds).getOrElse(fail("IO timed out during .accepted call")) match {
+        case Left(e)      => fail(s"IO failed when it was expected to succeed. Failure details: $e")
```

Comment:
$e.getMessage might be better, or there might be a `fail` overload that takes an exception?

Reply:
(`$e` might not have a good `.toString`)

Reply:
Done